daily_read/templates/daily_report.html

```diff
@@ -113,39 +113,56 @@ <h2><small>
 
 
         <div id="projects_list">
+          {% set priority = {1:['all_raw_data_delivered', 'All Raw Data delivered'], 2: ['all_samples_sequenced', 'All Samples sequenced'],
+                              3: ['qc_library_finished', 'Library QC finished'], 4: ['queued', 'Reception Control finished'],
+                              5: ['samples_received', 'Samples received']}
+           %}
+          {% for project in data['projects']%}
             <div class="panel panel-default">
                 <div class="panel-heading">
-                    <a data-toggle="collapse" href="#panel_project_a">
+                    <a data-toggle="collapse" href="#panel_project_{{ project['project_id'] }}">
                         <div class="row">
                             <div class="col-md-9">
-                                <h2 class="panel-title">Project A <span class="caret"></span></h2>
+                                <h2 class="panel-title">{{ project['project_name'] }} <span class="caret"></span></h2>
                             </div>
-                            <div class="col-md-3 text-right">Last updated: 2022-05-19</div>
+                            <!-- What should last updated be? -->
+                            <div class="col-md-3 text-right">Last updated: {{ data['pull_date'] }}</div>
                         </div>
                     </a>
                 </div>
-                <div class="panel-body collapse" id="panel_project_a">
-                    <h2>Sequencing started - 2022-05-19</h2>
-                    <h4>Library prepration started - 2022-04-31</h4>
-                    <h4>Samples received - 2022-04-14</h4>
-                </div>
-            </div>
-
-            <div class="panel panel-default">
-                <div class="panel-heading">
-                    <a data-toggle="collapse" href="#panel_project_b">
-                        <div class="row">
-                            <div class="col-md-9">
-                                <h2 class="panel-title">Project B <span class="caret"></span></h2>
-                            </div>
-                            <div class="col-md-3 text-right">Last updated: 2022-04-15</div>
-                        </div>
-                    </a>
-                </div>
-                <div class="panel-body collapse" id="panel_project_b">
-                    Panel content
+                <div class="panel-body collapse" id="panel_project_{{ project['project_id'] }}">
+                  <div class="progress">
```

Comment:
Rewrite progress bars and make test bigger too

Reply:
```suggestion
                  <!-- TODO: Rewrite progress bars and make text bigger too-->
                  <div class="progress">
```